src/vpn-editor/Openvpn/AdvancedSecurity.qml

```diff
@@ -55,6 +55,7 @@ Page {
                     i18n.tr("SEED-CBC"),
                     i18n.tr("AES-128-CBC-HMAC-SHA1"),
                     i18n.tr("AES-256-CBC-HMAC-SHA1"),
+                    i18n.tr("NONE"),
```

Comment:
If you add a new translation unit, you need to compile the project first locally, and then add the changes to the .pot file to your commit. Otherwise it cannot be translated.